server/src/notifications/notifications.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { NotificationsService } from './notifications.service';
+import { NotificationsController } from './notifications.controller';
+
+@Module({
+  providers: [NotificationsService],
+  controllers: [NotificationsController]
+})
```

Comment:
`controller`랑 `service`에서 `jwtService`랑 `prismaService` 사용하는 거 같은데 `imports` 없이도 동작하나요?

```suggestion
@Module({
  imports: [JwtModule, PrismaModule]
  providers: [NotificationsService],
  controllers: [NotificationsController]
})
```

Reply:
> `controller`랑 `service`에서 `jwtService`랑 `prismaService` 사용하는 거 같은데 `imports` 없이도 동작하나요?

fix 완료했습니다!!